man/systemd-stub.xml

```diff
@@ -94,16 +95,22 @@
       details on encrypted credentials. The generated <command>cpio</command> archive is measured into TPM
       PCR 4 (if a TPM is present)</para></listitem>
 
-      <listitem><para>Similar, files <filename><replaceable>foo</replaceable>.efi.extra.d/*.raw</filename>
-      are packed up as <command>cpio</command> archive and placed in the <filename>/.extra/sysext/</filename>
+      <listitem><para>Similarly, files <filename><replaceable>foo</replaceable>.efi.extra.d/*.raw</filename>
+      are packed up in a <command>cpio</command> archive and placed in the <filename>/.extra/sysext/</filename>
       directory in the initrd file hierarchy. This is supposed to be used to pass additional system extension
       images to the initrd. See
       <citerefentry><refentrytitle>systemd-sysext</refentrytitle><manvolnum>8</manvolnum></citerefentry> for
       details on system extension images. The generated <command>cpio</command> archive containing these
       system extension images is measured into TPM PCR 8 (if a TPM is present).</para></listitem>
+
+      <listitem><para>Files <filename>/loader/credentials/*.cred</filename> are packed up in a
+      <command>cpio</command> archive and placed in the <filename>/.extra/global_credentials/</filename>
+      directory of the initrd file hierarchy. This is supposed to be used to pass additional credentials to
+      the initrd, regardless of the kernel being booted. The generated <command>cpio</command> archive is
+      measured into TPM PCR 4 (if a TPM is present)</para></listitem>
```

Comment:
Could you add a separate para with a "synopsis" of the paths? Either at the top of the man page, or at the top of the "Companion Files" section. See e.g. sysusers.d(5) for an example. This way it'll be easier to see what paths are accepted at a glance, without needing to read a wall of text.

Reply:
The paths depend on the name and location of the binary. How should I handle that? How should I differentiate that some of the files referenced in the Synopsis are on the Linux rootfs and some are on the ESP?

For instance:
1. _ESP_/EFI/Linux/_foo_.efi.extra.d/*.cred
2. _ESP_/.../_foo_.efi.extra.d/*.cred
3. _foo_.efi.extra.d/*.cred
4. _foo_.efi.extra.d\\*.cred

Same with the global dir:
1. _ESP_/loader/credentials/*.cred
2. \loader\credentials\*.cred

Reply:
For paths on the ESP, we usually say something like `<filename><replaceable>ESP</replaceable>/EFI/BOOT/BOOT*.EFI</filename>`. For paths in the real file system, just specify a global path starting with `/`. And if anything else is necessary, please just make something up as you see fit. This idea is not to make this a formal specification, but rather to make it easy for a reader to remind themselves about the paths relevant to a typical installation.

Reply:
Done